v2/emailpassword/quick-setup/backend.mdx

```diff
@@ -51,11 +52,12 @@ supertokens.init({
     },
     appInfo: {
         // learn more about this on https://supertokens.io/docs/emailpassword/appinfo
+        // Compulsory
         appName: "YOUR APP NAME", // Example: "SuperTokens",
         apiDomain: "YOUR API DOMAIN", // Example: "https://api.supertokens.io",
         websiteDomain: "YOUR WEBSITE DOMAIN", // Example: "https://supertokens.io"
+        // Optional
         apiBasePath: "YOUR API BASE PATH", // Example: "/auth"
```

Comment:
Remove `apiBasePath` and `// Optional` from everywhere in the frontend and backend configs..